### PR TITLE
BC Gov Mines Act Permit Credential

### PR DESCRIPTION
**Pull request by PatStLouis**
This is a draft PR looking to align the BC Gov Energy and Mines Digital Trust framework with the traceability vocab to enable data exchanges. We are looking for feedback from both the traceability group on where these credentials would fit and the BC Gov EMDT governance team. 

[Here's the reference documentation](https://bcgov.github.io/bc-vcpedia/credentials/bc-mines-act-permit/1.1.1/governance.html)

### CODE REVIEW

File: docs/openapi/components/schemas/common/MinesActPermit.yml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    default:
+      - MinesActPermit
+    items:
+      type: string
+      enum:
+        - MinesActPermit
+  permit_no:
+    title: Permit Number
+    description: >-
+      (3-digit) Alpha-numeric identifier: M-##- or C-##- Permit Numbers are assigned to a specific mining operation or activity that has been granted permission or authorization. Mines Act Section 10 (1).
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/MinesActPermit.yml:23]: 3-digit, so shouldn't there be 3 numeric placeholders?
```suggestion
      (3-digit) Alpha-numeric identifier: M-###- or C-###- Permit Numbers are assigned to a specific mining operation or activity that has been granted permission or authorization. Mines Act Section 10 (1).
```

File: docs/openapi/components/schemas/common/MinesActPermit.yml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    description: >-
+      Name of the mining company that holds the permit.
+    type: string
+    $linkedData:
+      term: permittee_name
+      '@id': https://w3id.org/traceability#permittee_name
+  mine_operation_status:
+    title: Mine Operation Status
+    description: >-
+      Layer one in the hierarchy of identifying mine status. Mines status can be (1) Abandoned (2) Closed (3) Not Started (4) Operating.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/MinesActPermit.yml:47]:
```suggestion
      Layer one in the hierarchy of identifying mine status. Mine status can be (1) Abandoned (2) Closed (3) Not Started (4) Operating.
```

File: docs/openapi/components/schemas/common/MinesActPermit.yml [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    description: >-
+      (00.0000000) The geographic coordinate representing the north-south position of a specific location related to the administrative building at the mining operation or mine site.
+    type: string
+    $linkedData:
+      term: latitude
+      '@id': https://w3id.org/traceability#latitude
+  longitude:
+    title: Longitude
+    description: >-
+      (-000.0000000) The geographic coordinate that represents the east-west position of a specific location related to the administrative building at the mining operation or mine site.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/MinesActPermit.yml:111]: Either this change, or `representing` should change to `that represents`
```suggestion
      (00.0000000) The geographic coordinate representing the north-south position of a specific location related to the administrative building at the mining operation or mine site.
    type: string
    $linkedData:
      term: latitude
      '@id': https://w3id.org/traceability#latitude
  longitude:
    title: Longitude
    description: >-
      (-000.0000000) The geographic coordinate representing the east-west position of a specific location related to the administrative building at the mining operation or mine site.
```

File: docs/openapi/components/schemas/common/MinesActPermit.yml [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+    description: >-
+      (0.00) Lists the total bond value of the permit. A permittee is required to provide monetary security to cover reclamation costs, and to provide for the protection of, and mitigation of damage to, watercourses and cultural heritage resources affected by the mine.
+    type: string
+    $linkedData:
+      term: bond_total
+      '@id': https://w3id.org/traceability#bond_total
+  tsf_operating_count:
+    title: TSF Operating Count
+    description: >-
+      Numerical value for the total number of tailing storage facilities that are in Operating status.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/MinesActPermit.yml:127]:
```suggestion
      Numeric value for the total number of tailing storage facilities that are in Operating status.
```

File: docs/openapi/components/schemas/common/MinesActPermit.yml [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+    type: string
+    $linkedData:
+      term: tsf_operating_count
+      '@id': https://w3id.org/traceability#tsf_operating_count
+  tsf_care_and_maintenance_count:
+    title: TSF Maintenance Count
+    description: >-
+      Numerical value for the total number of tailing storage facilities that are in Care and Maintenance status.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/MinesActPermit.yml:135]:
```suggestion
      Numeric value for the total number of tailing storage facilities that are in Care and Maintenance status.
```

File: docs/openapi/components/schemas/common/MinesActPermit.yml [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+  - mine_disturbance
+  - mine_no
+  - issue_date
+  - latitude
+  - longitude
+  - bond_total
+  - tsf_operating_count
+  - tsf_care_and_maintenance_count
+example: |-
+  {
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/MinesActPermit.yml:158]: Please fill these up with sample values

File: docs/openapi/components/schemas/credentials/MinesActPermitCredential.yml [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+      - latitude
+      - longitude
+      - bond_total
+      - tsf_operating_count
+      - tsf_care_and_maintenance_count
+  proof:
+    type: object
+additionalProperties: false
+example: |-
+  {
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/credentials/MinesActPermitCredential.yml:132]: Similar to previous comment about examples

**Review by mkhraisha — CHANGES_REQUESTED** [review]

File: docs/openapi/components/schemas/common/MinesActPermit.yml (hunk 1 above)
Comment (PatStLouis) [reply docs/openapi/components/schemas/common/MinesActPermit.yml:23]: I will request an example to find out and confirm the exact form of the value.

**Comment by mkhraisha** [comment]
has there been any movement on this PR?

**Comment by nissimsan** [comment]
@PatStLouis , any updates on this?

**Comment by PatStLouis** [comment]
@nissimsan we are closing this PR as our efforts with the `MinesActPermit` will be focused on UNTP

**Pull request closed without merging**